### PR TITLE
Add cmake icon

### PR DESCRIPTION
**Pull request by Mikayex**
**Changes proposed:**
* [X] Add
* [ ] Prepare
* [ ] Delete
* [ ] Fix

**Things I've done:**
* [ ] My pull request fixes an issue, I referenced the issue.

The cmake icon is applied to these files:
- CMakeLists.txt
- CMakeCache.txt
- *.cmake

### CODE REVIEW

File: src/build/supportedExtensions.js [hunk 1]
```diff
@@ -45,6 +45,7 @@ exports.extensions = {
     { icon: 'coffeescript', extensions: ['coffee'] },
     { icon: 'config', extensions: ['env', 'ini', 'makefile', 'config', 'conf', 'cfg'] },
     { icon: 'config', extensions: ['makefile', '.env.example'], filename: true },
+    { icon: 'cmake', extensions: ['CMakeCache.txt', 'CMakeLists.txt', '.cmake'], filename: true, svg: true },
```
Comment (robertohuertasm) [inline src/build/supportedExtensions.js]: Can you order it alphabetically? 😉

Comment (Mikayex) [inline src/build/supportedExtensions.js]: Oops ! I thought it was but it seems I was tired...
Fixed !

**Comment by robertohuertasm** [comment]
@jens1o what's the problem?

**Comment by Mikayex** [comment]
@jens1o I've made another commit to fix the order : b3fe8675f11220b0927646076fe31109b369836d

**Comment by jens1o** [comment]
I just looked after the differences in the line deleted and the line added.... There was no difference between them. Until I realized the order itself had been changed ^^

**Comment by JimiC** [comment]
Guys, the ordering is trivial. The build process assures that the list is sorted.

**Comment by jens1o** [comment]
But it makes the file more readable, don't it?

**Comment by robertohuertasm** [comment]
It's for readability purposes.

**Pull request merged**